tests/phpunit/tests/actions.php

```diff
@@ -229,35 +229,133 @@ public function test_action_args_with_php4_syntax() {
 		$this->assertSame( array( $val ), array_pop( $argsvar ) );
 	}
 
-	public function test_action_priority() {
-		$a         = new MockAction();
+	/**
+	 * @ticket 60193
+	 *
+	 * @dataProvider data_action_priority_integers
+	 * @dataProvider data_action_priority_unhappy_path_nonintegers
+	 *
+	 * @covers ::do_action
```

Comment:
```suggestion
	 * @covers ::do_action
	 *
	 * @dataProvider data_action_priority_integers
	 * @dataProvider data_action_priority_unhappy_path_nonintegers
```

Reply:
Why is this order preferred?

Currently, there's not yet a coding standard for the order of these tests annotations. And there's no one predominant pattern within the test suite. There is a Trac ticket for specifying the test annotation order, but this has not yet been adopted.